USER_MANUAL.md

```diff
@@ -385,5 +385,12 @@ Manual steps:<br>
   1. Start the gym again and note that the level actors should be at their previous shutdown positions, not their original positions. You can do this repeatedly.
   1. The test has now passed. Don’t forget to revert the two settings changes you made before you run another test.
 
+##### Cross-Server possession gym
+* Functional tests the Controller remote possess the Pawn
+* How to test:
+  * Set the current map to the map which in the folder(/Game/Content/Maps/FunctionalTests/)  
+  * Set the `Multi Worker Settings Class` to `BP_Possession_Settings_Zoning2_2`(World Settings > Multi Worker)
```

Comment:
The point of adding new maps was so we don't have to make changes to the level setting prior to running the tests. Why is this necessary?